src/packages/BalancedJs/contracts/bnUSD.ts

```diff
@@ -72,4 +72,17 @@ export default class bnUSD extends Contract {
     });
     return this.callIconex(payload);
   }
+
+  public async transfer(toAddress: string, value: number): Promise<any> {
```

Comment:
I'd like to use the names of the same parameters as names of smart contract method params.
toAddress -> to